Packs/CheckpointFirewall/ReleaseNotes/2_0_8.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### CheckPointFirewall_v2
+- Added *ignore_warnings* and *ignore_errors* arguments to **checkpoint-host-add**.
```

Comment:
Added *ignore_warnings* and *ignore_errors* arguments to **checkpoint-host-delete** and **checkpoint-host-add**